packages/strapi-design-system/vite.config.ts

```diff
@@ -37,6 +38,6 @@ export default defineConfig(async () => {
         plugins: [typescript()],
       },
     },
-    plugins: [react()],
+    plugins: process.env.DTS !== 'true' ? [react()] : [dts({ entryRoot: 'src' }), react()],
```

Comment:
why do you have to declare `entryRoot`? we don't in the `primitives` package? :)

Reply:
Because otherwise it creates an src folder inside the dist folder with all the d.ts files

Reply:
Really? that's weird, I would write a comment to highlight this then, because it's not clear.

Reply:
and it is because of this line of code in the tsconfig   https://github.com/strapi/design-system/blob/1d6bc42b4f9a0798db39b25308fea68f8f86e9e1/packages/strapi-design-system/tsconfig.json#L19
```
"baseUrl": ".",
    "paths": {
      "@test/*": ["./test/*"]
    }
```

Reply:
> I would write a comment to highlight this then

see previous comment 👍🏼